jpos/src/test/java/org/jpos/iso/SslChannelIntegrationTest.java

```diff
@@ -65,7 +65,14 @@ public void serverSideDisconnect() throws Exception {
 
         XMLChannel clientChannel = newClientChannel();
 
-        clientChannel.connect();
+        while (!clientChannel.isConnected()) {
```

Comment:
This can loop forever and could hog the tests. I'd add a counter, and a small delay within the loop.